semeio/jobs/design2params/design2params.py

```diff
@@ -203,6 +203,14 @@ def _validate_design_matrix(design_matrix):
     if len(empties) > 0:
         raise SystemExit("Design matrix contains empty cells {}".format(empties))
 
+    # Look for initial or trailing whitespace in column headers. This
+    # is disallowed as it can create user confusion and has no use-case.
+    for col_header in design_matrix:
+        if col_header != col_header.strip():
+            raise SystemExit(
+                'Column header "{}" contains whitespace.'.format(col_header)
```

Comment:
Perhaps we should change the message to say that is has initial or trailing whitespaces? Would be good if the condition and the message corresponds ;)

Reply:
Yes. Good point!